### PR TITLE
feat(signalilo): add ingress support

### PR DESCRIPTION
**Pull request by pree**
#### What this PR does / why we need it:

* adds ingress support to the signalilo chart

#### Checklist

- [x] Chart Version bumped
- [x] I have run `make docs`
- [x] Variables are documented in the values.yaml using the format required by [Helm-Docs](https://github.com/norwoodj/helm-docs#valuesyaml-metadata).
- [x] PR contains the label that identifies the chart, e.g. `chart/<chart-name>`
- [x] PR contains the label that identifies the type of change, which is one of
      [ `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency` ]

### CODE REVIEW

**Review by simu — COMMENTED** [review]
Can you please rebase the PR on the latest master, that should fix the failing CI check.

Also, please note that we only recommend running Signalilo in the same cluster/network as the Alertmanager instance sending alerts to Signalilo. We've never tested a setup where Signalilo is not in the same local network as the Alertmanager instance connected to it.

**Comment by pree** [comment]
Thanks for the review @simu. I've added a note inside the `values.yaml`. Do you think this is sufficient, or should I add an disclaimer somewhere else?

**Review by simu — APPROVED** [review]
> Thanks for the review @simu. I've added a note inside the `values.yaml`. Do you think this is sufficient, or should I add an disclaimer somewhere else?

If you wouldn't mind, I'd like to also have the disclaimer in the README for the chart, other than that this LGTM now.

**Comment by pree** [comment]
Added also to the README of the Signalilo chart :)

File: appuio/signalilo/README.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ The README.md file is automatically generated with helm-docs!
 Edit the README.gotmpl.md template instead.
 -->
 
+## Deployment considerations
+
+It is recommended to deploy Signalilo only in the same cluster or network as the alertmanager instance, which is sending the alerts.
```
Comment (simu) [inline appuio/signalilo/README.md:23]:
```suggestion
It is recommended to deploy Signalilo only in the same cluster or network as the alertmanager instance which is sending the alerts.
```

**Comment by simu** [comment]
Thanks for the contribution :+1:

**Pull request merged**